src/main/java/Duke.java

```diff
@@ -1,10 +1,184 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    private static Task[] taskList = new Task[100];
+    private static String[] storeUserText = new String[100];
```

Comment:
Variable name should be a noun to differentiate from methods. For example, userText instead of storeUserText which sounds like a method.